src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -3766,17 +4225,107 @@ mod tests {
             assert!(test_bed.get_rat_position() <= 0.);
             assert!(test_bed.get_rat_rpm() <= 1.);
 
-            // Stopping both engines
+            // Killing AC buses
             test_bed = test_bed
-                .stop_eng1()
-                .stop_eng2()
+                .ac_bus_1_lost()
+                .ac_bus_2_lost()
                 .run_waiting_for(Duration::from_secs(2));
 
             // RAT has not deployed
             assert!(test_bed.get_rat_position() <= 0.);
             assert!(test_bed.get_rat_rpm() <= 1.);
         }
 
+        #[test]
+        fn rat_deploys_on_both_ac_lost() {
+            let mut test_bed = test_bed_with()
+                .set_cold_dark_inputs()
+                .in_flight()
+                .start_eng1(Ratio::new::<percent>(80.))
+                .start_eng2(Ratio::new::<percent>(80.))
+                .run_waiting_for(Duration::from_secs(10));
+
+            assert!(!test_bed.get_rat_deploy_command());
+
+            // Killing AC1 bus
+            test_bed = test_bed
+                .ac_bus_1_lost()
+                .run_waiting_for(Duration::from_secs(2));
+
+            // RAT has not deployed
+            assert!(!test_bed.get_rat_deploy_command());
+
+            //Now all AC off
+            test_bed = test_bed
+                .ac_bus_1_lost()
+                .ac_bus_2_lost()
+                .run_waiting_for(Duration::from_secs(2));
+
+            // RAT avail
+            assert!(test_bed.get_rat_deploy_command());
```

Comment:
**suggestion**: As above.

Reply:
General note: the `test_bed` structure was set up so that tests describe themselves very well without using comments. Comments are only needed when describing some edge-case for which you have to provide functional information to the dev.